crowbar_framework/app/models/backup.rb

```diff
@@ -172,6 +176,7 @@ def save_archive
     self.version = meta["version"]
     self.size = path.size
     self.created_at = Time.zone.parse(meta["created_at"])
+    self.migration_level = meta["migration_level"] || ActiveRecord::Migrator.last_migration.version
```

Comment:
> so that we come to a state of cloud6 without updates?

(not sure what you mean)

> what would happen with that hard coded value when we add another migration?

Well, that's where the second call to migrate is used :-) This is what happens:

- option a: backup doesn't have this migration level metadata. It means that it was created before this PR got merged, which means that the migration level is the hardcoded one I suggest. (It cannot be lower than that due to the fact that the backup feature was non-existing before)

- option b: backup has this metadata. It means our first call to migrate moves us to this level, we do the restore, then we call migrate to the latest level.

I'm not sure if what I'm saying is clear. Does that make sense?